server/graphql/v1/CollectiveInterface.js

```diff
@@ -1849,6 +1852,19 @@ const CollectiveFields = () => {
         return collective;
       },
     },
+    hasTwoFactorAuth: {
+      type: GraphQLBoolean,
+      async resolve(collective) {
+        const user = await models.User.findOne({
+          where: { CollectiveId: collective.id },
+        });
+        if (user.twoFactorAuthToken) {
```

Comment:
This will crash if we try to fetch this field on something else than a user collective. For example with an org, `models.User.findOne` will return `null`